src/main/java/seedu/address/logic/parser/UnFavoriteCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.UnFavoriteCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FavoriteCommand object
+ */
+public class UnFavoriteCommandParser implements Parser<UnFavoriteCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FavoriteCommand
+     * and returns an FavoriteCommand object for execution.
```

Comment:
Should it be the context of the UnFavoriteCommand ...  returns an UnFavoriteCommand object...?

Reply:
Good catch! Updated ✔